helm/defectdojo/templates/initializer-job.yaml

```diff
@@ -10,6 +10,9 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     helm.sh/chart: {{ include "defectdojo.chart" . }}
+    annotations:
+      "helm.sh/hook": post-install,post-upgrade
```

Comment:
Can we keep hook outside of default, as part of jobAnnotations inputs?

Reply:
it seems like we can :) wasn't sure because its a annotation for HELM itself, but the generated templates look good.